packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -24,14 +36,32 @@ export function NativeValue({ data, edit, onClick, onChange, jsonpath }) {
 	if (edit) {
 		return <input type={inputType} value={data} onChange={e => onChange(e, { jsonpath })} />;
 	}
+
+	const isSelectedLine = (selectedJsonpath && (selectedJsonpath === jsonpath));
```

Comment:
question why not simply `const isSelectedLine = selectedJsonpath === jsonpath;`

Reply:
The point is to avoid having true for each row when those two are valued to undefined or null